minify.sh

```diff
@@ -8,14 +11,18 @@ fi
 cd "$(dirname $0)"
 
 cat > $output <<-'EOF'
-#!/bin/bash -e
+#!/bin/bash
+set -x
+set -e
+
 dest=${1:-gentoo-build}
-mkdir $dest
 if ! sed --version 2>&1 | grep -q 'GNU sed'; then
   SEDCMD=gsed
 else
   SEDCMD=sed
 fi
+
+mkdir $dest
```

Comment:
Is there a reason for moving  the `mkdir` here? (If there is no reason, then lets keep it the old way)

Reply:
No, just separate "variables set" phase. 
Feel free to change

Reply:
@OndrejHome , done.